### PR TITLE
[ES-86800] Add Dockerfiles for CUDA 11

### PR DESCRIPTION
**Pull request by panchalhp-db**
As titled

# Testing
Tested these images by uploading them to by own dockerhub @ panchalhp and starting clusters for:
- [pytorch](https://e2-dogfood.staging.cloud.databricks.com/?o=6051921418418893#notebook/2937713537498489/command/2937713537498494)
 
Screen Shot 2021-04-29 at 14 53 04

- [tensorflow](https://e2-dogfood.staging.cloud.databricks.com/?o=6051921418418893#notebook/2937713537499698/command/2937713537499699)
 
Screen Shot 2021-04-29 at 14 53 33

### CODE REVIEW

**Review by lu-wang-dl — APPROVED** [review]

File: ubuntu/gpu/cuda-11/conda/Dockerfile
Comment (sarthfrey) [inline ubuntu/gpu/cuda-11/conda/Dockerfile:1]: `databricksruntime` to replace `panchalhp`

File: ubuntu/gpu/cuda-11/pytorch/Dockerfile
Comment (sarthfrey) [inline ubuntu/gpu/cuda-11/pytorch/Dockerfile:1]: `databricksruntime` to replace `panchalhp`

File: ubuntu/gpu/cuda-11/rapids/Dockerfile
Comment (sarthfrey) [inline ubuntu/gpu/cuda-11/rapids/Dockerfile:1]: `databricksruntime` to replace `panchalhp`

File: ubuntu/gpu/cuda-11/tensorflow/Dockerfile
Comment (sarthfrey) [inline ubuntu/gpu/cuda-11/tensorflow/Dockerfile:1]: `databricksruntime` to replace `panchalhp`

**Review by sarthfrey — CHANGES_REQUESTED** [review]
woot cuda 11! I remember a year back I had to actually train a model with pytorch/tensorflow to make sure those libraries functioned correctly in the DCS container, instead of just checking that gpus were accessible, but if you're confident they work to full extent then perfect :) I just left some comments to replace your dockerhub name

File: ubuntu/gpu/cuda-11/conda/Dockerfile
Comment (panchalhp-db) [reply ubuntu/gpu/cuda-11/conda/Dockerfile:1]: Yep, I'll do that once I have push access to databricksruntime. I'll have to chat with Evan about getting permission. Will change this before merging it :)

**Review by sarthfrey — APPROVED** [review]
Approving as my comments are minor :)

**Comment by panchalhp-db** [comment]
> woot cuda 11! I remember a year back I had to actually train a model with pytorch/tensorflow to make sure those libraries functioned correctly in the DCS container, instead of just checking that gpus were accessible, but if you're confident they work to full extent then perfect :) I just left some comments to replace your dockerhub name

@sarthfrey We have DUST Suites that test the same behavior that I tested so I am confident that this should work. MLR 8.0+ have these versions of TF and pytorch working with CUDA 11 so I don't think there should be any issue.

**Comment by sarthfrey** [comment]
> > woot cuda 11! I remember a year back I had to actually train a model with pytorch/tensorflow to make sure those libraries functioned correctly in the DCS container, instead of just checking that gpus were accessible, but if you're confident they work to full extent then perfect :) I just left some comments to replace your dockerhub name
> 
> @sarthfrey We have DUST Suites that test the same behavior that I tested so I am confident that this should work. MLR 8.0+ have these versions of TF and pytorch working with CUDA 11 so I don't think there should be any issue.

Ah okay great, LGTM 🚢

**Pull request merged**
